UnityProject/Assets/Scripts/Health/CPRable.cs

```diff
@@ -74,14 +74,15 @@ private void ServerDoCPR(GameObject performer, GameObject target, BodyPartType T
 				{
 					if (organ is Lungs lung)
 					{
-
+						lung.TryBreathing(node, 1, true);
+						lung.TryBreathing(node, 1, true);
```

Comment:
...why can't you just increase the amount of air taken in? calling this 3 times seems bad